packages/cloudcasa/package.yaml

```diff
@@ -1,2 +1,2 @@
-url: https://github.com/catalogicsoftware/cloudcasa-helmchart/releases/download/cloudcasa-1.0/cloudcasa-1.0.tgz 
+url: https://github.com/catalogicsoftware/cloudcasa-helmchart/releases/download/cloudcasa-1.0.0/cloudcasa-1.0.0.tgz
```

Comment:
@PennyScissors can you confirm that this is okay?

Reply:
I got confirmation from the partner that this is fine